src/provider/local-path.lisp

```diff
@@ -293,6 +294,33 @@ GROUP-MIGRATION-FILES-BY id function."
                    :up-script-path nil
                    :down-script-path file-path)))
 
+(defgeneric touch-migration (kind local-path-provider description &key id-generating-function)
+  (:documentation "Creates an empty up-and-down pair of migration resources using the given local-path-provider. If KIND is :lisp, for cl-migratum to invoke your lisp migration handler function, please replace :my-system, :my-package, :my-handler-function in the newly created pair of files as your need."))
```

Comment:
Please wrap the documentation to multiple lines, similarly to the other generic function docstrings.

Reply:
Sure. Thanks.

Reply:
Fixed in [0864759](https://github.com/dnaeon/cl-migratum/pull/16/commits/08647596223b60d0b7cc7f1e6bf2d2942da1379f)